src/routers/alpha-router/alpha-router.ts

```diff
@@ -1957,16 +1917,28 @@ export class AlphaRouter
 
       quotePromises.push(
         Promise.all([v3CandidatePoolsPromise, v2CandidatePoolsPromise]).then(
-          ([v3CandidatePools, v2CandidatePools]) =>
-            this.mixedQuoter
+          async ([v3CandidatePools, v2CandidatePools]) => {
```

Comment:
why async now compared to before?

Reply:
because we are now fetching the crossLiquidityCandidatePools, which returns a promise